src/component/Text.tsx

```diff
@@ -190,6 +189,12 @@ const getWordsByLines = ({ width, scaleToFit, children, style, breakAll, maxLine
 
 const DEFAULT_FILL = '#808080';
 
+const reduceCSSCalc = (calc: string) => {
+  const tempEle = document.createElement('div');
```

Comment:
In how far is this temporary, if you create it on every call?

Reply:
> In how far is this temporary, if you create it on every call?

Can place the tempEle outside of the function scope to avoid creating it in every call